consensus/cometbft/service/log/sdk_logger.go

```diff
@@ -23,26 +23,25 @@ package log
 import (
 	sdklog "cosmossdk.io/log"
 	"github.com/berachain/beacon-kit/log"
+	"github.com/berachain/beacon-kit/log/phuslu"
 )
 
-type SDKLogger[LoggerT log.AdvancedLogger[LoggerT]] struct {
-	log.AdvancedLogger[LoggerT]
+type SDKLogger struct {
```

Comment:
Todo: check if we can drop this type and what interface it is implementing (to assert it)